### ISSUE TITLE
Web Service Sender: "Error caching WSDL. Please check the WSDL URL and authentication settings" when click on Get Operations of WSDL URL

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#1871]
When I configure a Web Service Sender Connector, populate the correct WSDL URL and click on "Get Operations" button, after a few seconds operating, mirth shows a pop-up with the message "Error caching WSDL. Please check the WSDL URL and authentication settings".

The URL is reachable and all configuration is correct. With Mirth v2.0.1 works fine. I think that the problem is in authentication. My webservice required authentication, but when I populate the correct user and pass, it doesn't work. However, If I delete the WS authentication, it works fine.

Versions affected: 2.1.0 and 2.1.1

I think the problem is in the ws extension because I've managed to install ws 2.0.1 extension in Mirth 2.1.0 and works fine. But I don't like this patch.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-1895
Reporter: sergiogda
Created: 2011-07-14T00:40:57.000-0700

**Comment by rbeckman-nextgen** [comment]
Do you have a public example wsdl that we can reproduce this problem with?

Imported Comment. Original Details:
Author: jacobb
Created: 2011-08-22T13:59:34.000-0700

**Comment by rbeckman-nextgen** [comment]
Waiting on an example WSDL that causes this issue.

Imported Comment. Original Details:
Author: jacobb
Created: 2011-11-10T13:33:44.000-0800

**Comment by rbeckman-nextgen** [comment]
I have the same problem:
https://apis.medimo.nl/koppelvlak/helios/heliostest/mirth.php?wsdl
above URL can be used.
I'm using Mirth Connect Server 2.2.0.5828

Imported Comment. Original Details:
Author: bigbee31
Created: 2012-04-02T05:06:14.000-0700

**Comment by rbeckman-nextgen** [comment]
Testing with Mirth 2.2.1.5861, with the URL at medimo.nl there seems to be no problem getting operations, either with or without authentication (though auth isn't required for that URL).

Imported Comment. Original Details:
Author: rwhelan
Created: 2012-04-05T18:37:56.000-0700

**Comment by rbeckman-nextgen** [comment]
Unfortunately we will need a public web service that can reproduce this issue in order to fix it.  Please create a new issue if the problem still exists.

Imported Comment. Original Details:
Author: jacobb
Created: 2013-10-17T17:38:16.000-0700
